R/fct_findAdducts.R

```diff
@@ -76,13 +76,13 @@ annotateGrp <- function(ipeak, imz, rules, mzabs, devppm, isotopes, quasimolion,
     return(NULL);
   };
   
-  #Entferne Hypothesen, welche gegen OID-Score&Kausalität verstossen!
+  #Entferne Hypothesen, welche gegen OID-Score&Kausalitaet verstossen!
```

Comment:
Bonus points for translating german comments ...

Reply:
:) I leave that to you :) I just removed to german characters because `devtools::document()` failed crying for non-UTF8 encoding.